### PR TITLE
add img alt

### PR DESCRIPTION
**Pull request by Petrie**

### CODE REVIEW

**Comment by svrnm** [comment]
Thanks for your contribution @Petrie. One change request: those images are not really logos just icons/pictograms representing those languages, can you remove the word "logo" from all `alt` tags, e.g.:

```html
  <img width="35"
  src="https://raw.github.com/open-telemetry/opentelemetry.io/main/iconography/32x32/C++_SDK.svg"
  alt="C++"></img>
```

**Comment by Petrie** [comment]
@svrnm Got that!

File: content/en/docs/instrumentation/net/_index.md [hunk 1]
```diff
@@ -1,7 +1,9 @@
 ---
 title: .NET
 description: >
-  <img width="35" src="https://raw.github.com/open-telemetry/opentelemetry.io/main/iconography/32x32/.NET.svg" alt="NET logo"></img>
+  <img width="35"
+  src="https://raw.github.com/open-telemetry/opentelemetry.io/main/iconography/32x32/.NET.svg"
+  alt="NET"></img>
```
Comment (Kielek) [inline content/en/docs/instrumentation/net/_index.md:6]:
```suggestion
  alt=".NET"></img>
```
or dotnet.

**Review by lalitb — APPROVED** [review]
Looks good for C++.

**Review by Kielek — APPROVED** [review]

File: content/en/docs/instrumentation/js/_index.md [hunk 2]
```diff
@@ -1,7 +1,9 @@
 ---
 title: JavaScript
 description: >-
-  <img width="35" src="https://raw.github.com/open-telemetry/opentelemetry.io/main/iconography/32x32/JS_SDK.svg" alt="JS logo"></img>
+  <img width="35"
+  src="https://raw.github.com/open-telemetry/opentelemetry.io/main/iconography/32x32/JS_SDK.svg"
+  alt="JS"></img>
```
Comment (svrnm) [inline content/en/docs/instrumentation/js/_index.md:6]: JS -> JavaScript

**Review by svrnm — CHANGES_REQUESTED** [review]

**Review by reyang — APPROVED** [review]

File: content/en/docs/instrumentation/erlang/_index.md [hunk 3]
```diff
@@ -2,7 +2,9 @@
 title: Erlang/Elixir
 weight: 14
 description: >
-  <img width="35" src="https://raw.github.com/open-telemetry/opentelemetry.io/main/iconography/32x32/Elixir_SDK.svg"></img>
+  <img width="35"
+  src="https://raw.github.com/open-telemetry/opentelemetry.io/main/iconography/32x32/Elixir_SDK.svg"
+  alt="Erlang"></img>
```
Comment (tsloughter) [inline content/en/docs/instrumentation/erlang/_index.md:7]:
```suggestion
  alt="Erlang/Elixir"></img>
```

**Review by nrcventura — APPROVED** [review]

**Review by pjanotti — APPROVED** [review]
.NET looks good.

**Review by srikanthccv — APPROVED** [review]

File: content/en/docs/instrumentation/erlang/_index.md [hunk 4]
```diff
@@ -2,7 +2,9 @@
 title: Erlang/Elixir
 weight: 14
 description: >
-  <img width="35" src="https://raw.github.com/open-telemetry/opentelemetry.io/main/iconography/32x32/Elixir_SDK.svg"></img>
+  <img width="35"
+  src="https://raw.github.com/open-telemetry/opentelemetry.io/main/iconography/32x32/Elixir_SDK.svg"
```
Comment (svrnm) [inline content/en/docs/instrumentation/erlang/_index.md:6]: can you use the Erlang logo here? (should just be s/Elixir/Erlang) cc: @tsloughter

**Review by svrnm — APPROVED** [review]

**Comment by svrnm** [comment]
I think this looks good overall, thank you @Petrie

**Pull request merged**